cli/nix/udf-wrapper.nix

```diff
@@ -0,0 +1,18 @@
+{ pkgs, ... }:
+
+{
+  packages.udf-wrapper = pkgs.buildNpmPackage {
```

Comment:
Does it use the build script defined in `package.json`?

Reply:
I think it does, yes